### DISCUSSION TITLE
Nullable and Filled

### DISCUSSION THREAD

**Discussion by leonardocustodio** [rebing/graphql-laravel#951] [Q&A]
Shouldn't a field with the following rules:
`['nullable', 'filled']`

Be able to work if:
- Not present
- Is null
- Anything but not empty

Right now if we pass the filled rule, the nullable one doesn't work if we pass null explicitly.
We get the error `The field must have a value.`

**Comment by mfn** [comment]
Those Laravel validation rules combinations always give me headaches 🤯

So, are you sure any kind of behaviour you observed is specific to _this_ library and not how Laravel itself behaves? I suggest you perform some tests without GraphQL context to make sure.

**Reply by leonardocustodio** [reply]
I can't say I'm sure :laughing: 
I will make some tests and let you know, thanks ;D
